target/linux/ath79/dts/qca9558_belkin_f9j1108-v2.dts

```diff
@@ -0,0 +1,231 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca955x.dtsi"
+
+/ {
+	model = "Belkin AC1750 DB Wi-Fi (F9J1108v2)";
+	compatible = "belkin,f9j1108-v2", "qca,qca9558";
+
+	aliases {
+		led-boot = &led_system;
+		led-failsafe = &led_system;
+		led-running = &led_system;
+		led-upgrade = &led_system;
+		label-mac-device = &eth1;
+	};
+	
+	leds {
+		compatible = "gpio-leds";
+
+		led_system: system {
+			label = "belkin:blue:system";
```

Comment:
Typically (there are exceptions), the model part should be used for the first part here:
f9j1108-v2:blue:system

Reply:
according to ``Documentation/devicetree/bindings/leds/common.yaml``:
>   label:
>>    description:
>>>      The label for this LED. If omitted, the label is taken from the node name
>>>      (excluding the unit address). It has to uniquely identify a device, i.e.
>>>      no other LED class device can be assigned the same label. This property is
>>>      deprecated - use 'function' and 'color' properties instead.
>>>      function-enumerator has no effect when this property is present.

see the deprecation info